flowershop/models/product.py

```diff
@@ -0,0 +1,45 @@
+from odoo import fields,models,api
+import collections
+
+class Product(models.Model):
+    _inherit = "product.product"
+
+    is_flower=fields.Boolean(string="Is flower?",
+    )
+    flower_ids=fields.Many2one('flowershop',
```

Comment:
```suggestion
    flower_id = fields.Many2one('flowershop',
```

TIP: naming convention here as well:
id -> many2one
ids -> one2many, many2many